Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/ContactDiaryOverviewViewModel.kt

```diff
@@ -16,7 +19,8 @@ import org.joda.time.LocalDate
 import timber.log.Timber
 
 class ContactDiaryOverviewViewModel @AssistedInject constructor(
```

Comment:
Do we really need this here(changes in this class)?
>There is also an Worker who already starts the cleaning..
maybe cleanup would be good

Reply:
Yes, accodding to tech spec we must provide solution to clean obsolete data without background workers.

Reply:
true